addon/kubeflow.v1.2.0/manifest.yaml

```diff
@@ -227,6 +253,8 @@ spec:
           value: /login/oidc
         - name: oidc_auth_url
           value: /dex/auth
+        - name: skip_auth_uri
+          value: "/dex /.well-known"
```

Comment:
I am just curious, why is there a space between `dex` and `/.well-known`?

Reply:
https://github.com/ajmyyra/ambassador-auth-oidc/blob/master/README.md#options

>SKIP_AUTH_URI Space separated whitelist of URIs like "/info /health" to bypass authorization.

Reply:
This means no matter `TLS` is enabled or not, `/.well-known` will always be in the allowlist.
Can we add an if statement around this, so we can make sure `/.well-known` is only needed when TLS is enabled?

Reply:
it doesn't matter, because we don't have anything in that directory, but it's easy to add one more check here

Reply:
done